pkg/agent/agent.go

```diff
@@ -137,10 +138,42 @@ func flowsAgent(cfg *Config,
 	exporter node.TerminalFunc[[]*flow.Record],
 	agentIP net.IP,
 ) (*Flows, error) {
-	// configure allow/deny interfaces filter
-	filter, err := initInterfaceFilter(cfg.Interfaces, cfg.ExcludeInterfaces)
-	if err != nil {
-		return nil, fmt.Errorf("configuring interface filters: %w", err)
+	var filter InterfaceFilter
+
+	if len(cfg.InterfaceIPs) > 0 {
```

Comment:
do we need to flag an error if user supplied both `cfg.InterfaceIPs` and `cfg.Interfaces` ?